doc/sphinx/language/core/sections.rst

```diff
@@ -1,3 +1,6 @@
+Section Mechanism
+====================================
+
 .. _section-mechanism:
 
 Section mechanism
```

Comment:
```suggestion
Sections
```

"Mechanism" doesn't add anything and "Sections" is more similar to the titles of the other subsections.